db/migrate/20160219202933_create_incurred_incidentals.rb

```diff
@@ -0,0 +1,12 @@
+class CreateIncurredIncidentals < ActiveRecord::Migration
+  def change
+    create_table :incurred_incidentals do |t|
+      t.references :incidental_type, index: true, foreign_key: true
+      t.decimal :times_modified
+      t.text :notes
+      t.text :document
```

Comment:
We probably will want to change document to be a separate model instead of a single field, I imagine a check-in officer would want to upload multiple images showing different angles on the same damage